### ISSUE TITLE
Does teamcity-slack support svn as a repository?

### ISSUE THREAD

**Issue by matfricker** [alexkvak/teamcity-slack#115]
Based on the following question on your trouble-shooter guide:
Q: I checked the option to send private messages and added the {mention} placeholder to the message, but neither the message was send to the slack user nor the name was mentioned in the slack message!

I was wondering if svn is supported? Our svn server is hooked up via windows authentication and the user / author that is used is a name and not the email address.

Do you have any suggestions?

**Comment by alexkvak** [comment]
Plugin uses TeamCity API and gets the user and other VCS information from this API.

So does TeamCity recognize the commit users in build history? Are they shown in the UI?

**Comment by matfricker** [comment]
Yes, TeamCity does recognise commit users, but the commit username is not an email address. 

I do have an email address in my TeamCity profile and username is not an email address.

Is the email address case-sensitive? My slack profile email address starts with a capital letter.

**Comment by alexkvak** [comment]
It does not matter. The VCS commit author should be associated with TeamCity user.

About case-sensitivity – yes it can be an issue. Try to use same case in TeamCity and Slack user info

**Comment by matfricker** [comment]
I can confirm that, email address in teamcity and slack are both the same case now. Have not received a DM message on fail, but the {mention} does have my handle. So it is picking up the user.

**Comment by alexkvak** [comment]
Is `Personal notifications enabled` enabled?

![изображение](https://user-images.githubusercontent.com/1052418/96602294-89079980-12fb-11eb-83de-038c1d144d41.png)

**Comment by matfricker** [comment]
Yes that is enabled.

**Comment by alexkvak** [comment]
If commit author's email cannot be found in Slack, the plugin should log this as error. 

Please take a look at teamcity-server.log in Administration -> Diagnostics -> Server Logs

**Comment by matfricker** [comment]
Apologies for the delay in responding...

I have checked server logs and can see an entry there that a message was sent to the correct email address.

I guess I was expecting something to popup in my direct messages but it doesn't. I can see my handle when the build fails under Status: failed. I get notifications from the channel, but all notifications (Started, Success & Failures).
